packages/kr.rb

```diff
@@ -0,0 +1,54 @@
+require 'package'
+
+class Kr < Package
+  description 'Krypton command-line client, daemon, and SSH integration'
+  homepage 'https://krypt.co'
+  version '2.4.4'
+  source_url 'https://raw.githubusercontent.com/kryptco/kr/master/README.md'
+  source_sha256 'a0452783d22218a6fab914abbf5263459aaa1c8745512d7c4ebcf51fe08aaf7a'
+
+  depends_on "rust" => :build
+  depends_on "go" => :build
+  depends_on "libsodium" => :build
+  depends_on "rsync" => :build
+  depends_on "psmisc" => :build
+
+  def self.install
+    cargo_web_sha256 = "c8a92ed0185206ad7f2da8e1692d168b20044de7066b2eb086e4bbb0d06ff787"
+    ENV["GOPATH"] = "#{Dir.pwd}/go"
+    ENV["RUSTUP_HOME"] = "#{Dir.pwd}/.rustup"
+    ENV["CARGO_HOME"] = "#{Dir.pwd}/.cargo"
+    system "mkdir -p #{ENV["RUSTUP_HOME"]}"
+    system "mkdir -p #{ENV["CARGO_HOME"]}"
+
+    ENV["PATH"] = "#{ENV["CARGO_HOME"]}/bin:/usr/local/bin:/usr/bin:/bin:/opt/bin"
```

Comment:
Why do you need `#{ENV["CARGO_HOME"]/bin` in the path?

Reply:
Removed it, I checked and all works without it. In trying to get this up and running I had come across something that indicated the need for that for cargo to work. So I had that in my path so I was just trying to replicate that for the sandboxed version. More evidence of me not knowing anything about rust and cargo. Thanks for catching that.